yinstruments/pdu/README.md

```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
 PDUs. The PDU class is the parent class to the Netbooter and Lindy. All of these 
 classes have the following callable functions: get_status, reboot, off, and on.
 
```

Comment:
Can you add two subsections here:

## Package
  * briefly mention/show how to import and instance your the pdu classes in the package

## CLI
  * Here you can keep the text below that talks about *SETUP:* and python environments.